core/math/color.h

```diff
@@ -169,14 +169,14 @@ struct _NO_DISCARD_ Color {
 		return res;
 	}
 
-	_FORCE_INLINE_ Color to_linear() const {
+	_FORCE_INLINE_ Color srgb_to_linear() const {
 		return Color(
 				r < 0.04045f ? r * (1.0 / 12.92) : Math::pow((r + 0.055f) * (float)(1.0 / (1 + 0.055)), 2.4f),
 				g < 0.04045f ? g * (1.0 / 12.92) : Math::pow((g + 0.055f) * (float)(1.0 / (1 + 0.055)), 2.4f),
 				b < 0.04045f ? b * (1.0 / 12.92) : Math::pow((b + 0.055f) * (float)(1.0 / (1 + 0.055)), 2.4f),
 				a);
 	}
-	_FORCE_INLINE_ Color to_srgb() const {
+	_FORCE_INLINE_ Color linear_to_srgb() const {
```

Comment:
The same change should be done in the `godot-cpp` as well - https://github.com/godotengine/godot-cpp/blob/60037decc36e223fbcce78586a26e7dd9468033d/include/godot_cpp/variant/color.hpp#L174-L186

Reply:
Done: https://github.com/godotengine/godot-cpp/pull/738